packages/account/src/Components/poi/status/unsupported/unsupported.tsx

```diff
@@ -34,11 +42,11 @@ const Unsupported = ({ country_code, handlePOIforMT5Complete, ...props }) => {
         return (
             <DetailComponent
                 is_onfido_supported={country_code === 'ng' && !checkNimcStep(documents[detail].details.documents)}
-                country_code={country_code}
+                country_code_key={country_code}
                 document={documents[detail]}
                 root_class='manual-poi'
                 onClickBack={() => setDetail(null)}
-                handlePOIforMT5Complete={handlePOIforMT5Complete}
+                handlePOIforMT5Complete={() => handlePOIforMT5Complete?.()}
```

Comment:
could you please explain, why do we need this changes? why just not pass function as before?
types for these components will look more complicated :)

Reply:
for handlePOIforMT5Complete, the function that is triggering might be undefined, so typescript will give error I pass in the function straight into the props. The logic here is to check whether the function is undefined only the function will be trigger.